frontend/app/src/people/widgetViews/WidgetSwitchViewer.tsx

```diff
@@ -103,7 +113,7 @@ function WidgetSwitchViewer(props: any) {
     offer: peopleOffers
   };
 
-  const activeList = [...listSource[selectedWidget]].filter(({ body }: any) => {
```

Comment:
is `listsource` being used anymore? if not we might want to remove it from this file

Reply:
removed